packages/react-server-webpack-plugin/package.json

```diff
@@ -7,7 +7,8 @@
   "types": "dist/index.d.ts",
   "sideEffects": false,
   "scripts": {
-    "build": "tsc --p tsconfig.build.json",
+    "build": "tsc --p tsconfig.build.json && yarn run copy-templates",
```

Comment:
This feels a bit hacky to me and won't scale well when we migrate to TypeScript project references (see https://github.com/Shopify/quilt/pull/962). Instead, I think we should tweak our configs a bit.

* Add `"./src/templates/*"` to the `tsconfig.json#include`
* Remove `"**/*/templates/**/*""` from the `tsconfig.json#exclude`
* Add `// @ts-ignore` to where the build fails since these are template files. If we could opt the whole file out of typechecking that would be better imo.

Reply:
I think this plan makes sense